### ISSUE TITLE
No output 

### ISSUE THREAD

**Issue by ShalokShalom** [rnestler/reboot-arch-btw#182]
Hi there. 

Reading the [source](https://github.com/rnestler/reboot-arch-btw/blob/master/src/checks.rs) lets me believe, that reboot-arch-btw is supposed to always put out a string. 

I currently get only an empty prompt from it. 
As I am trying to implement reboot-arch-btw into my own script, is testing this way a bit difficult. 

Am I right in the assumption, that reboot-arch-btw is supposed to always say *something*?

**Comment by rnestler** [comment]
Hi in #178 we added a `--verbose` flag which needs to be passed if you always want output.

**Comment by rnestler** [comment]
For some debug output you can additionally  set `RUST_LOG=info`.

**Comment by rnestler** [comment]
I updated the README in #183 to mention the `--verbose` flag.

**Comment by ShalokShalom** [comment]
> Making it more usable for scripting is probably a good goal. Would a different exist status help if a reboot / session restart is necessary?

With the current wording, I can simply match "reboot" and "restart" ☺️

**Comment by ShalokShalom** [comment]
With restart the session, do you mean the DE?

**Comment by rnestler** [comment]
> With restart the session, do you mean the display manager?

Yeah this is what I mean. On my system I use the following alias: `rab: aliased to reboot-arch-btw --reboot-packages systemd,linux-firmware,amd-ucode --session-restart-packages xorg-xwayland,sway --verbose`. And I usually then just logout when sway got updated when I'm done.

The session time is read from `/var/run/utmp` (see https://github.com/rnestler/reboot-arch-btw/blob/master/src/session.rs) and I don't think `plasmashell --replace` will update this.

**Comment by ShalokShalom** [comment]
Oh thanks, very helpful. `--session-restart-packages` simply does a logout?

**Comment by ShalokShalom** [comment]
It seems like it simply issues the text output. Thanks

**Comment by rnestler** [comment]
> Oh thanks, very helpful. --session-restart-packages simply does a logout?
> It seems like it simply issues the text output. Thanks

Yes the tool doesn't take any actions, it just informs if certain packages got upgraded.
